catroid/src/main/res/values/strings.xml

```diff
@@ -1167,15 +1164,15 @@
     <string name="brick_stop_script">Stop</string>
     <string name="brick_stop_this_script">this script</string>
     <string name="brick_stop_all_scripts">all scripts</string>
-    <string name="brick_stop_other_scripts">other scripts of this object</string>
+    <string name="brick_stop_other_scripts">other scripts of this actor or object</string>
 
 
     <plurals name="second_plural">
         <item quantity="one">"second "</item>
         <item quantity="other">"seconds "</item>
     </plurals>
 
-    <string name="brick_collision_receive">When physical collision with</string>
+    <string name="brick_collision_receive">When collision under gravity with</string>
```

Comment:
```suggestion
    <string name="brick_collision_receive">When you collide with</string>
```
This would be an even better change, *if* we can extend the functionality to actually also check collision among non physical objects, which should be just a few lines of code since it already works with various sensors and the generalizes "When .... becomes true" event brick. I think I will propose that as a "bug ticket" to get it handled quickly, as it really is not much, just a little bit of logic glue code is required, and the current "When physical collision" brick is very difficult to use and understand by our users, mainly by the unclear formulation and the limited functionality. Changing both the text and the code would allow to change the translations in one swoop, too. Important IMHO, and a low hanging fruit. The "you" in the text also fits well with the "When you receive" bricks etc.

Reply:
```suggestion
    <string name="brick_collision_receive">When you bounce off</string>
```
Even better and more precise. Please use this. The functionality also will be slightly adapted, but this string is already better than what we have had so far.